css/style.css

```diff
@@ -4,8 +4,232 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    margin: 0px;
+    padding: 0px;
 }
+   /*Header*/
+ .header{background-color: rgb(248, 248, 248);}
```

Comment:
As best practises, always start every css style on a new line? Line 11 should be written as:

`.header{`
`  background-color: rgb(248, 248, 248);`
`}`